CHANGELOG-draft.md

```diff
@@ -42,5 +42,17 @@ THIS FILE ACCUMULATES THE RELEASE NOTES FOR THE UPCOMING RELEASE.
 
 ## Internal changes
 
+* Integration test script now displays output interactively (#1700)
+* Fixed a few issues with error response documentation in Swagger (#1707)
+* Make mapping between (team) permissions and roles more lenient (#1711)
+* The `DELETE /conversations/:cnv/members/:usr` endpoint rewritten to Servant (#1697)
+* Remove leftover auto-connect internal endpoint and code (#1716)
+* Clean up JSON golden tests (#1729, #1732)
+
+## Federation changes
+
+* Added client certificate support for server to server authentication (#1682)
+* Implemented full server-to-server authentication (#1687)
+* Add an endpoint for removing a qualified user from a local conversation (#1697)
```

Comment:
This looks wrong

Reply:
Can you be a bit more specific?

Reply:
Those "added" lines didn't look like they belonged to this PR, apologies for being vague. But it looks like you solved it :+1:

Reply:
Do you want to update your review or do I still need to make some changes?

Reply:
I haven't looked at all the things, this was the only thing I found until the point I looked. I have dismissed the review. I will take a look once the conflicts are resolved.

Reply:
@akshaymankar , I just resolved the conflicts!

Reply:
Marking this as resolved.